### DISCUSSION TITLE
how to contribute to this repository?

### DISCUSSION THREAD

**Discussion by cannot-hero** [vuejs/language-tools#4095] [Q&A]
I clone this repo, and try to improve some new feature. But I dont know how to debug this code.
I use vscode, i tried F5 in my vscode, it did open a new extension development host, but the debugger or console some info in this repo's code doesnt work.
So can anybody tell me how to debug this? Thanks a lot! with picture or video is better

**Comment by yeongsheng-tan** [comment]
> I clone this repo, and try to improve some new feature. But I dont know how to debug this code. I use vscode, i tried F5 in my vscode, it did open a new extension development host, but the debugger or console some info in this repo's code doesnt work. So can anybody tell me how to debug this? Thanks a lot! with picture or video is better

Please check [here](https://github.com/vuejs/language-tools?tab=readme-ov-file#contribution-guide)

**Reply by cannot-hero** [reply]
thanks for your reply, I followed this but it seems still not work
